onnxruntime/core/providers/dml/OperatorAuthorHelper/MLOperatorAuthorHelper.h

```diff
@@ -10,6 +10,7 @@
 #define ML_CHECK_BOOL(x) ORT_THROW_HR_IF(E_INVALIDARG, !(x))
 #else
 #define ML_CHECK_BOOL(x) THROW_HR_IF(E_INVALIDARG, !(x))
+#define ML_CHECK_BOOL_MSG(x, msg) THROW_HR_IF_MSG(E_INVALIDARG, !(x), msg)
```

Comment:
>ML_CHECK_BOOL_MSG [](http://example.com/codeflow?start=8&length=17)

If adding to one side of an `ifdef`, add it to the other branch too (otherwise the code will break when ORT_NO_EXCEPTIONS is defined, and Epic will be unhappy).